CHANGELOG.md

```diff
@@ -7,11 +7,46 @@ Platform support legends:
     * Note: the papercuts will majorly be with the build process. Things might be very rough to integrate as no polish at all has been given yet.
 * ❌ = tier 3 support. It doesn't work just yet, but we plan to make it work.
 
+## [1.0.0-rc.6] - 2023-08-08
+
+<details>
+    <summary>git-conventional changelog</summary>
+
+### Bug Fixes
+
+- `e2eiRotateAll` return type was not wrapped
+- Signature KeyPair was rotated when credentials were which was zealous. Also fixes an important bug caused by inverted private & public keypair part when rotating credentials
+
+### Features
+
+- [**breaking**] Handle the case when a client tries to decrypt a Welcome referring to a KeyPackage he already has deleted locally
+- Add keystore dump exporter CLI tool
+
+### Testing
+
+- Add a roundtrip test for e2ei credential rotation to tackle a false positive regression
+
+</details>
+
+* Add keystore dump command line to debug internal applications and export the content of the keystore for further analysis
```

Comment:
```suggestion
* Add keystore dump CLI tool to debug internal applications and export the content of the keystore for further analysis
```

Reply:
Oops, discard that, this needs to be added to the .tpl file and regenerate the .md afterwards!